<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacher_sel</class>
 <widget class="QDialog" name="teacher_sel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="sel_buto">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>显示信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ret">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>160</width>
     <height>320</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>工号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>联系方式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>出生日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>身份证号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>民族</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>工作地点</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>所在班级</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>备注</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>160</width>
     <height>326</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="id"/>
    </item>
    <item>
     <widget class="QLineEdit" name="name"/>
    </item>
    <item>
     <widget class="QLineEdit" name="sex"/>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneNum"/>
    </item>
    <item>
     <widget class="QLineEdit" name="birth"/>
    </item>
    <item>
     <widget class="QLineEdit" name="idNum"/>
    </item>
    <item>
     <widget class="QLineEdit" name="nation"/>
    </item>
    <item>
     <widget class="QLineEdit" name="work"/>
    </item>
    <item>
     <widget class="QLineEdit" name="class_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="remarks"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
